tests/ReactionsApiTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+use Rogue\Models\Photo;
+
+class ReactionsApiTest extends TestCase
+{
+    /*
+     * Base URL for the Api.
+     */
+    protected $reactionsApiUrl = 'api/v2/reactions';
+
+    /**
+     * Test that the POST /reactions request creates a reaction for a reactionable.
+     * Also test that when POST /reactions is hit again by the same user for the same reactionable,
+     * the reaction is soft deleted.
+     *
+     * POST /reactions
+     * @return void
+     */
+    public function testPostingAndSoftDeleteForReaction()
+    {
+        // Create a photo to react to.
+        $photo = factory(Photo::class)->create();
+
+        $reaction = $this->createReactionRequest(str_random(24), $photo->id, 'photo');
+
+        $this->json('POST', $this->reactionsApiUrl, $reaction);
+
+        $this->assertResponseStatus(200);
```

Comment:
@katiecrane ah for some reason I thought that `201` response codes are for updates! Since it (should be) a small update, I can do this in this PR too?